inlong-sort/sort-connectors/iceberg/src/main/java/org/apache/inlong/sort/iceberg/FlinkConfigOptions.java

```diff
@@ -22,6 +22,9 @@
 import org.apache.flink.configuration.ConfigOption;
 import org.apache.flink.configuration.ConfigOptions;
 
+/**
+ * Iceberg unique flink configuration
+ */
 public class FlinkConfigOptions {
 
     public static final ConfigOption<Boolean> TABLE_EXEC_ICEBERG_INFER_SOURCE_PARALLELISM =
```

Comment:
TABLE_EXEC_ICEBERG_INFER_SOURCE_PARALLELISM
TABLE_EXEC_ICEBERG_INFER_SOURCE_PARALLELISM_MAX
they are not being used in current code, is it nessasary to keep them ?

Reply:
ok, remove this class